### PR TITLE
ENH Mechanism for sign up instructions + privacy page (#456)

### PR DESCRIPTION
**Pull request by rth**
Backport of #456 to master. Partially addresses https://github.com/paris-saclay-cds/ramp-board/issues/440

Should have no immediate impact when deployed to ramp.studio instance, if only the links to slack would need to be put in the `config.yaml`, instead of them being hardcoded.

### CODE REVIEW

**Comment by agramfort** [comment]
could the doc make explicit the way it is (should be) done ramp.studio as an example.

it's easy for me to update an example for my usecase. Filling blanks from scratch is harder.

my 2c

thanks so much @rth for taking the time to do this.

File: ramp-utils/ramp_utils/template/ramp_config_template.yml [hunk 1]
```diff
@@ -9,11 +9,15 @@ ramp:
     # predictions_dir: (absolute path where to store predictions. Default: <deployment_dir>/events/event_name>/predictions)
     # logs_dir: (absolute path where to store logs. Default: <deployment_dir>/events/<event_name>/logs)
     # sandbox_dir: (name of the default submissions. Default: starting_kit)
+    # privacy_policy_page: (path or HTML contents of the privacy policy page. Default: None)
+    # login_instructions: (path or HTML contents of custom instuctions to add to the login page. Default: None)
+    # sign_up_instructions: (path or HTML contents of custom instuctions to add to the login page. Default: None)
+    # sign_up_ask_social_media: (whether add folm field to ask for social media accounts on the sign up page. Default: False)
```
Comment (maikia) [inline ramp-utils/ramp_utils/template/ramp_config_template.yml:15]: did you mean form field?

File: doc/personalize.rst [hunk 2]
```diff
@@ -22,3 +22,42 @@ Your images must have one of the following extensions:
...
+directly the HTML contents of that page.
+
+This will enable the ``/privacy_policy`` endpoint, and will add it to the footer
+menu.
+
+
+Sign up and login pages
+-----------------------
+
+You can add personalized messages to the Sign Up and Login pages, as follows,
```
Comment (maikia) [inline doc/personalize.rst:49]: maybe for this section good to repeat where it has to be added (main
`config.yaml`)?

**Review by maikia — APPROVED** [review]
nice
thanks @rth

**Comment by rth** [comment]
I think I have addressed all comments, and improved the documentation a bit more. LGTM is a false positive.

**Pull request merged**